app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="sort_by">Sort By</string>
```

Comment:
This should be removed.

Reply:
You're right I removed it!
Yes, I followed the way you implemented the other options and then I only needed to find the right queries.